resources/ts/Components/StatusLight.tsx

```diff
@@ -33,35 +32,9 @@ export const StatusLight = ({ status }: Props) => {
           {t(`integrations.status.${status}`)}
         </div>
       </div>
-      {(status === "pending_approval_payment" ||
-        status === "pending_approval_integration") && (
-        <div className="flex flex-col gap-3">
-          <div>
-            {status === "pending_approval_integration" ? (
-              <span>
-                {t(
-                  "integrations.status.pending_approval_integration_description"
-                )}
-              </span>
-            ) : (
-              <>
-                <span>
-                  {t(
-                    "integrations.status.pending_approval_payment_description"
-                  )}
-                </span>
-                <Link className="text-publiq-blue-dark" href="#">
-                  {" "}
-                  {t("integrations.status.here")}
-                </Link>
-              </>
-            )}
-          </div>
-          <ButtonPrimary className="self-start">
-            {t("integrations.status.activate")}
-          </ButtonPrimary>
-        </div>
-      )}
+      {["pending_approval_integration", "pending_approval_payment"].includes(
```

Comment:
It could be safer to use the `Status.PENDING_APPROVAL_...` constant here. Less brittle that just strings